### ISSUE TITLE
No package metadata found

### ISSUE THREAD

**Issue by VladasZ** [TimNN/cargo-lipo#61]
I got this errror on latest `cargo` versions:
```shell
cargo lipo --release
[ERROR cargo_lipo] No package metadata found for "path+file:///Users/user/crate_name#0.1.0"
```
Probably something has changed in packages naming.

I patched it like this: https://github.com/VladasZ/cargo-lipo/commit/a63452528816d5124503fb0fd6ce3b5eb33d6450
It works fine for me but I'm not sure if this is a proper fix.

**Comment by TimNN** [comment]
Thanks for the report, I'll try to get this fixed next weekend.

**Comment by TimNN** [comment]
Should be fixed in `v3.3.0`.

**Comment by VladasZ** [comment]
Yeah everything works fine now. Thank you.
